src/libsync/discovery.cpp

```diff
@@ -1897,4 +1904,61 @@ void ProcessDirectoryJob::setupDbPinStateActions(SyncJournalFileRecord &record)
         record._type = ItemTypeVirtualFileDownload;
 }
 
+void ProcessDirectoryJob::processCaseClashConflicts(const PathTuple &path,
+                                                    LocalInfo &localEntry,
+                                                    [[maybe_unused]] const RemoteInfo &serverEntry,
+                                                    const SyncJournalFileRecord &dbEntry,
+                                                    const std::map<QString, Entries> &entries) const
+{
+    for(const auto &entry : entries) {
```

Comment:
Using a structured binding here instead of `entry` would make this easier to read IMO:

```suggestion
    for(const auto &[key, entry] : entries) {
```

would save from having to call `entry.second` repeatedly (and also better match the fact that `entry.second` is the actual entry